res/TensorFlowLiteRecipes/Inf_Mul_000/test.recipe

```diff
@@ -0,0 +1,30 @@
+operand {
```

Comment:
plz split this to another PR

Reply:
Ah, then I'll mark this PR as a draft and create separate PRs.